NSS-Day-Cohort-53/tabloid-full-stack-always-puffy-impalas
# View Posts

#### Changes Made
1. Added Post module
2. Added PostList module
3. Added postManager module
4. Added Post model
5. Added Tag model
6. Added Category model
7. Added PostRepository and its interface with GetPublishedPosts method
8. Added Post Controller
9. Added routing to "/posts"
10. Added link to post list in nav bar
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cg-viewPosts
    ```
2. Open a new Terminal tab and navigate to the server directory.
3. Test app functionality.
     > npm start to run app
     > before logging in, notice that you may not navigate to /posts when unauthorized
     > log in
     > use nav bar link to route to /posts
     > observe list of posts generated
 
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.

Closes #45
